src/lib/recoil/assessment.js

```diff
@@ -104,10 +104,50 @@ export const personalityAssessmentQuery = selectorFamily({
   key: "assessment/personality"
 });
 
+export const genericAssessmentQuery = selectorFamily({
```

Comment:
Since we're now going from a survey oriented object to a results object, we might need to check the state a little more.

Right now I get some errors when I run through a fresh assessment. Once I click "Finish" it fully breaks:

- Warning: Failed prop type: The prop `assessmentResult.responses` is marked as required in `Score`, but its value is `undefined`.
- Warning: Failed prop type: The prop `assessmentResult.responses` is marked as required in `Breakdown`, but its value is `undefined`.
- Uncaught TypeError: can't access property "length", assessmentResult.responses is undefined
- Uncaught TypeError: can't access property "map", assessmentResult.responses is undefined

I think those components just need to better check if the results are ready